tilt-provider.json

```diff
@@ -9,7 +9,8 @@
           "go.mod",
           "go.sum",
           "api",
-          "internal",
+          "controllers",
```

Comment:
Have we tried running Tilt with these changes? I assume it was already broken before this PR (I can only see a single `pkg` folder in the repo which is in the root), but would be good to fix it with the same change since we are touching it anyway

Reply:
i have not, i just made sure things compiled. not sure i even know what change you'd like me to make? change `controllers` to `bootstrap/controllers` ?

Reply:
@luthermonson sorry for not making it clear, I was referring to `"pkg"` which does not exist inside of neither https://github.com/luthermonson/cluster-api-provider-rke2/tree/drop-internal/bootstrap and https://github.com/luthermonson/cluster-api-provider-rke2/tree/drop-internal/controlplane.

It is not a blocker for this PR and can be done (if it makes sense!) in a follow-up as well.